README.md

```diff
@@ -89,9 +89,9 @@ docker run -d \
   makuk66/docker-solr:4.10.4
 ```
 
-#### Build open-oni
+#### Build Open ONI
 
-Start the development open-oni.  This will install requirements if needed, and
+Start the development Open ONI  This will install requirements if needed, and
```

Comment:
Probably should keep the period here.  And maybe add "container", since "the development Open ONI" sounds sort of weird.